### PR TITLE
Fix old tokens missing after migration

### LINKED ISSUE #3335: Old tokens are gone
The `db/migrate/20170621083718_add_st_ito_tokens.rb` migration added STI to the Token class but it did not change old data. So old tokens don't know who they are now...

### PR DESCRIPTION
**Pull request by mdeniz**
It fixes #3335

### CODE REVIEW

File: src/api/db/migrate/20170630144825_convert_tokens_to_service_tokens.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ConvertTokensToServiceTokens < ActiveRecord::Migration[5.1]
+  def up
+    Token.where(type: nil).update_all(type: 'Token::Service')
+  end
+
+  def down
+    Token::Service.update_all(type: nil)
```
Comment (ChrisBr) [inline src/api/db/migrate/20170630144825_convert_tokens_to_service_tokens.rb]: is down really necessary? As when we revert this migration we drop the field anyway?
But it is fine for me to leave it, just as a comment!

Comment (mdeniz) [inline src/api/db/migrate/20170630144825_convert_tokens_to_service_tokens.rb]: Just to leave the things as they were?... Not super-mandatory, but anyway to be consistent

Comment (bgeuken) [inline src/api/db/migrate/20170630144825_convert_tokens_to_service_tokens.rb]: If it's supposed to be the inverse of 'up' it should be

```ruby
Token.where(type:  'Token::Service').update_all(type: nil)
```

**Comment by bgeuken** [comment]
LGTM,

but as mentioned I would keep the down part empty

File: src/api/db/migrate/20170630144825_convert_tokens_to_service_tokens.rb (hunk 1 above)
Comment (hennevogel) [inline src/api/db/migrate/20170630144825_convert_tokens_to_service_tokens.rb]: I agree. Let's remove it

**Pull request merged**